sympy/combinatorics/fp_groups.py

```diff
@@ -207,7 +207,8 @@ def order(self, strategy="relator_based"):
         2
 
         """
-        from sympy import S, gcd
+        from sympy import gcd
```

Comment:
Can we use symengine here?

Reply:
This would specifically require `gcd_list` function from `SymPy`.